src/app/shared/table/actions/cars/table-create-car-action.ts

```diff
@@ -22,8 +23,8 @@ export class TableCreateCarAction extends TableCreateAction {
   }
 
   private createCar(carDialogComponent: ComponentType<unknown>,
-    dialog: MatDialog, refresh?: () => Observable<void>) {
-    super.create(carDialogComponent, dialog, null, refresh, {
+    dialog: MatDialog, dialogParams: DialogParamsWithAuth<Car, CarAuthorizationActions>, refresh?: () => Observable<void>) {
```

Comment:
The goal to introduce AuthorizationActions was to have only one type of param for create dialog so I would have expected only this param: `dialogParams: AuthorizationActions`